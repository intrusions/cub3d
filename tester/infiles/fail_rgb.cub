NO ./texture/no.xpm
SO ./texture/so.xpm
WE ./texture/we.xpm
EA ./texture/ea.xpm
F  0a,0,0
C 225,30,0,

111111
1E0001
111111
