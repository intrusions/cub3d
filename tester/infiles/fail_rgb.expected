Loading scene...
Error
	fail_rgb.cub: Invalid RGB: C
	fail_rgb.cub: Invalid RGB: F
	fail_rgb.cub: Incorrect field: C
Quitting...
